COPYRIGHT

```diff
@@ -0,0 +1,48 @@
+Intellectual Property Notice
+------------------------------
+
+CARE is licensed under the BSD-3 license,
+(BSD-3 or https://opensource.org/licenses/BSD-3-Clause).
+
+See the LICENSE file for details.
+
+Copyrights and patents in the CARE project are retained by contributors.  No
+copyright assignment is required to contribute to CARE.
+
+
+SPDX usage
+------------
+
+Individual files contain SPDX tags instead of the full license text.
+This enables machine processing of license information based on the SPDX
+License Identifiers that are available here: https://spdx.org/licenses/
+
+Files that are licensed as BSD-3 contain the following
+text in the license header:
+
+    SPDX-License-Identifier: BSD-3 Clause
+
+
+External Packages
+-------------------
+CARE has some external dependencies, some of which are included as Git
+submodules. These packages are covered by various permissive licenses. 
+A summary listing follows. See the license included with each package
+for full details.
+
+PackageName: Umpire
+PackageHomePage: https://github.com/LLNL/Umpire
+PackageLicenseDeclared: MIT
+
+PackageName: BLT
+PackageHomePage: https://github.com/LLNL/blt/
+PackageLicenseDeclared: BSD-3-Clause
+
+PackageName: RAJA
+PackageHomePage: https://github.com/LLNL/raja/
+PackageLicenseDeclared: BSD-3-Clause
+
+PackageName: CHAI
+PackageHomePage: https://github.com/LLNL/chai/
+PackageLicenseDeclared: MIT
+
```

Comment:
You should add in the new submodules here.

Reply:
Done